### PR TITLE
feat: GEOT-218: create group EPIC

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

File: jest.coverage.ignore.js [hunk 1]
```diff
@@ -10,13 +10,16 @@ const PERMANENT_IGNORE = [
...
+  './packages/app-andrew/src/pages/group-create/index.tsx',
   './packages/app-andrew/src/server.ts',
-  './packages/ui/SavedSearch/index.tsx',
+  './packages/app-main/src/components/rules/AcceptRulesButton/index.tsx',
   './packages/app-main/src/pages/rules/index.tsx',
   './packages/app-main/src/providers/Rules/index.tsx',
-  './packages/app-main/src/components/rules/AcceptRulesButton/index.tsx',
+  './packages/ui/PostSingle/index.tsx',
+  './packages/ui/SavedSearch/index.tsx',
+  './packages/ui/SearchBar/index.tsx',
```
Comment (ra8ga) [inline jest.coverage.ignore.js:21]: this must be removed

File: jest.coverage.ignore.js [hunk 2]
```diff
@@ -10,13 +10,16 @@ const PERMANENT_IGNORE = [
...
+  './packages/ui/SearchResults/index.tsx',
```
Comment (ra8ga) [inline jest.coverage.ignore.js:22]: SavedSearch and SearchResults - these were removed

Comment (mariusz-kabala) [reply jest.coverage.ignore.js:22]: so remove it from ignore list as well

**Comment by mariusz-kabala** [comment]
retest this please

File: jest.coverage.ignore.js (hunk 2 above)
Comment (ra8ga) [reply jest.coverage.ignore.js:22]: removed here https://github.com/gtms-org/gtms-frontend/pull/479/files

**Pull request merged**